app/models/order.rb

```diff
@@ -29,6 +30,22 @@ class Order < ActiveRecord::Base
 
   scope :active_orders, -> { where('status NOT IN (?)', INACTIVE_STATUS) }
 
+  def update_packages
```

Comment:
It would be good if this function name reflected more about what it is doing. e.g.  `designate_orders_packages`